easybuild/easyconfigs/c/CGAL/CGAL-4.14.3-gompi-2021a.eb

```
@@ -28,7 +28,7 @@ dependencies = [
     ('Qt5', '5.15.2'),
 ]
 
-configopts = "-DOPENGL_INCLUDE_DIR=$EBROOTMESA/include\;$EBROOTLIBGLU/include "
+configopts = '-DOPENGL_INCLUDE_DIR="$EBROOTMESA/include;$EBROOTLIBGLU/include" '
```

consistency across `CGAL` easyconfigs
```suggestion
configopts = r"-DOPENGL_INCLUDE_DIR=$EBROOTMESA/include\;$EBROOTLIBGLU/include "
```

---

There is no need for the escape when quoting the value. This is actually correct as the original (unquoted) would break if `$EBROOTMESA` contains spaces

---

in which case, most of the other `CGAL` easyconfigs should be updated since they use `r"-DOPENGL_INCLUDE_DIR=$EBROOTMESA/include\;$EBROOTLIBGLU/include "`

---

True. Done